context-free.md

```diff
@@ -2,21 +2,45 @@
 
 ##  Who is this programming language for?
 
+Computer-aided drawing of images
 
 ## What is easy to do in this language? Why is it easy?
 
+Making basic shapes, and to some extend making lines ad paths, are easy. Some
+forms of randomness, but not the kind that I would have preferred to use, are
+also easy. Modifying these shapes is also very easy, for certain types of
+modifications. These things are easy bcause they are supported by the built-in
+library and primitive constructs of the language.
 
 ## What is hard to do in this language? Why is it hard?
 
+Some forms of randomness, like in expressions instead of choosing rules, are not
+only hard but impossible. Other things, like animation, are hard to work with,
+but could potentially be worked around. These are both hard because they are not
```

Comment:
Working 'around' language features may subvert the purpose of language, even if you are unhappy with the original language features.